unit-tests/src/test/scala/java/util/function/BiConsumerTest.scala

```diff
@@ -25,14 +30,12 @@ class BiConsumerTest {
     assertTrue(result == 4)
   }
 
-  @Ignore("andThen doesn't work currently")
   @Test def biConsumerAndThen(): Unit = {
-    // result = 0
-    // assertTrue(result == 0)
-    // addT.andThen(timesT).accept(2, 3)
+    assertTrue(result == 0)
+    addT.andThen(timesT).accept(2, 3)
 
-    // // addT will add 2 + 2 = 4 to result which is zero = 4
-    // // then 2*3 = 6 will be added to result which is 4, =  11
-    // assertTrue(result == 11)
+    // addT will add 2 + 2 = 4 to result which is zero = 4
```

Comment:
I suggest deleting the comment lines.  What was it that Neils Bohr said about a theory that was so off that it was not even wrong?    The arguments given in this case are 2 and 3, not 2 and 2.  So the sum is 5, added to the (2 * 3 === 6) gives the expected 11.
By my reading, the assert is right and the comment is total misdirection; a polite way of saying so wrong as to waste people's time.

Reply:
Thank you for studying, fixing, & enabling the AndThen test.  It adds assurance that the javalib code is working.

Reply:
Ok, I can fix that but I can't fix everything.

Reply:
Understood. The value of the whole is much better than the individual parts.  
Thank you for removing that comment.  It was so far off that it surely would have
snagged somebody or somebodies, probably us in 6 month time.

Reply:
Sure, no problem. It was pretty much wrong - I was just happy the tests passed. I have tried to clean things up a bit but have tried to just do a direct port as to not confuse matters.